pwndbg/heap/ptmalloc.py

```diff
@@ -24,11 +34,23 @@
 import pwndbg.gdblib.typeinfo
 import pwndbg.gdblib.vmmap
 import pwndbg.glibc
+import pwndbg.heap
+import pwndbg.heap.heap
+import pwndbg.lib.cache
+import pwndbg.lib.memory
 import pwndbg.search
 from pwndbg.color import message
 from pwndbg.color.memory import c as M
 from pwndbg.constants import ptmalloc
-from pwndbg.heap import heap_chain_limit
+
+if typing.TYPE_CHECKING:
```

Comment:
I don't think it's obvious what's going on here, can you add a comment explaining it?

Reply:
It's a dance to avoid importing `pwndbg.heap.structs` since it's imported dynamically when first resolving `HeuristicHeap.struct_module`. I'll add a comment too.